### PR TITLE
Add the pe_histogram tool

### PR DESCRIPTION
**Pull request by gregvonkuster**

### CODE REVIEW

File: tools/pe_histogram/pe_histogram.xml [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+        <!-- Anything other than zero is an error -->
+        <exit_code range=":-1" />
+        <exit_code range="1:" />
+        <!-- Check stderr in case the return code has not been set -->
+        <regex match="Error:" />
+        <regex match="Exception:" />
+    </stdio>
+    <command>
+        <![CDATA[
+            ln -f -s "${input_bam.metadata.bam_index}" "${input_bam}.bai" &&
```
Comment (bgruening) [inline tools/pe_histogram/pe_histogram.xml]: Greg you are lining the bai file into the database dir and not into the current working dir, isn't it?
You should rather link input_bam and the index into cwd.

Comment (gregvonkuster) [inline tools/pe_histogram/pe_histogram.xml]: @bgruening Good catch - I believe I have fixed this here: https://github.com/galaxyproject/tools-iuc/pull/817/commits/753c6ed43dfc3568990f2ee60c4f4253ef7136dc

**Comment by gregvonkuster** [comment]
Can I merge this PR even though the tests failed?

**Comment by gregvonkuster** [comment]
This tool has been uploaded to both the TTS and MTS.

**Pull request merged**